### ISSUE TITLE
Report to swap parameters when literals or enum members are placed on left side of comparison

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#2206]
We should report to swap the parameters or arguments when the constant values (literals, enum members) are placed on the left side.

Example:
```c#
if (StringComparison.Ordinal.Equals(someValue)) { ... }
```
should be changed into:
```c#
if (someValue.Equals(StringComparison.Ordinal)) { ... }
```

Same is valid for:
```c#
if (StringComparison.Ordinal == someValue) { ... }
```
should be changed into:
```c#
if (someValue == StringComparison.Ordinal) { ... }
```

**Comment by RalfKoban** [comment]
For the comparisons using `==` or other we already have rule [MiKo_3084](https://github.com/RalfKoban/MiKo-Analyzers/blob/master/Documentation/MiKo_3084.md).
So we only need to extend it for the `Equals` comparisons.
